.github/workflows/release.yml

```diff
@@ -66,4 +66,25 @@ jobs:
           author: asyncapi-bot <info@asyncapi.io>
           title: 'chore(release): ${{ steps.extractver.outputs.version }}'
           body: 'Version bump in package.json and package-lock.json for release [${{ steps.extractver.outputs.version }}](https://github.com/${{github.repository}}/releases/tag/v${{ steps.extractver.outputs.version }})'
-          branch: version-bump/${{ steps.extractver.outputs.version }}
\ No newline at end of file
+          branch: version-bump/${{ steps.extractver.outputs.version }}
+  
+  dispatch:
+    needs: release
+    strategy:
+      matrix:
+        repo: ['asyncapi/generator']
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - id: package_details
```

Comment:
we do not need this custom action. We already have this script `npm run get-version` and we can add script that echos `process.env.npm_package_name` too.

so eventually this step will look like this:
```
      - name: Get version and name from package.json
        id: package_details
        run: |
          echo "::set-output name=version::$(npm run get-version --silent)"
          echo "::set-output name=name::$(npm run get-name --silent)"
```

Reply:
I actually like the action better. We can publish this action instead of having it here and then we would not need to touch every package.json in every project we want to use it.

I mean, having something like this:

```
      - name: Get version and name from package.json
        id: package_details
        uses: fmvilas/package-details@v1
```

Don't you think it's better? No need to add custom scripts to package.json.

Reply:
I'm always in position to create custom action, or use community action only when it is really really necessary and saves time (this is why in generator docker release is not done with any special action).

I suggested npm scripts as they are anyway already in use in all the release workflows in all repos. I just prefer to stick to one. If you insist on custom action, then sure but have code separated from this repo, and we need to make sure it is used everywhere and stop using npm scripts.

Also, better than custom action would be to contribute here https://github.com/EndBug/version-check to get then expose package name to output. We already use this action here and in html-template to identify if there is minor or patch release, so we could make it a standard and get rid of npm scripts

Reply:
It's the same effort for me to add another package.json script to get the package name and replace the current methodology with another action. I'd go for my suggestion, really. Happy to contribute to the action you mentioned but it's really a stupid action and there are a few like the one you mentioned. I'd use my action first and contribute to that package. If they don't accept the PR (and it would be expected because it's called `version` check) at least we have it working.